src/resources/js/components/layout/SideBar.vue

```diff
@@ -35,7 +56,32 @@ export default {
         path: '/presentation/creer',
       },
     ],
+    notAuthRoutes: [
+      {
+        name: 'Connexion',
+        icon: 'mdi-account-circle',
+        path: '/login',
+      },
+    ],
+    isLoaded: false,
+    isLogged: false,
   }),
+  beforeMount() {
+    window.axios.get('/me')
+    // i need a way to know it's a success, but I don't actually care about the response
+    // eslint-disable-next-line no-unused-vars
+      .then((response) => {
+        // handles success
+        this.isLogged = true;
+      })
+      .catch(() => {
+        // handles error
+        this.isLogged = false;
+      })
+      .then(() => {
+        this.isLoaded = true;
+      });
```

Comment:
following the axios workflow, this will never be executed

Reply:
it actually is